algebra/include/matrix.h

```diff
@@ -31,6 +31,14 @@ extern int matrix_bufAlloc(matrix_t *matrix, int rows, int cols);
 extern void matrix_bufFree(matrix_t *matrix);
 
 
+/* return a pointer to matrix element in specified row and column
+ * if row or column exceeds matrix size NULL is returned */
+inline float *matrix_at(matrix_t *A, unsigned int row, unsigned int col)
```

Comment:
```suggestion
inline float *matrix_at(const matrix_t *A, unsigned int row, unsigned int col)
```

There is little const correctness in `matrix` module of `libalgeb`, but better start now than never :)

Reply:
Done